### ISSUE TITLE
omz plugin list: command not found (column)

### ISSUE THREAD

**Issue by platon-p** [ohmyzsh/ohmyzsh#11254]
### Describe the bug

**Version**: `master (b9be3a4)`
**OS**: Ubuntu 22.04.1 LTS

When I'm using command `omz plugin list`, it raises errors like

```
Custom plugins:
_omz::plugin::list:13: command not found: column

Built-in plugins:
_omz::plugin::list:20: command not found: column
```

### Steps to reproduce

Try to complete `omz plugin list`
P.S. I didn't install any plugins

### Expected behavior

```
Custom plugins:
_omz::plugin::list:13: command not found: column

Built-in plugins:
_omz::plugin::list:20: command not found: column
```

### OS / Linux distribution

Ubuntu 22.04.1 LTS

### Zsh version

5.8.1

### Oh My Zsh version

master (b9be3a4)

### Terminal emulator

iTerm (but I'm using ssh)

**Comment by mcornella** [comment]
Run `sudo apt install bsdextrautils` for Ubuntu 22.04. However this is a bug if `column` is not supposed to be installed by default.

**Comment by platon-p** [comment]
I think it should be installed as dependency by default, if it is not installed

**Comment by rwmitchell** [comment]
column has been a standard Unix cmd for a very long time.

**Comment by platon-p** [comment]
It turns out that there is no 🤔

**Comment by mcornella** [comment]
> I think it should be installed as dependency by default, if it is not installed

OMZ does not install dependencies as that would require sudo access and it wouldn't be expected by the user either. As `column` is just a text formatting tool we can remove the dependency altogether or just display a long list if it's not installed.

> column has been a standard Unix cmd for a very long time.

There may be systems were it is not installed it they're not normally used interactively. For example I have confirmed that the Docker image for Ubuntu does not have it, and it could be the same for Ubuntu Server edition.

In any case, this is an easy fix, no need to debate it.

**Comment by mcornella** [comment]
Reopening to keep track of it until we fix the dependency.

**Comment by mcornella** [comment]
Fixed in [8487a55](https://github.com/ohmyzsh/ohmyzsh/commit/8487a5536d247927216dd22041c1c32bcd100256).
